database/test.go

```diff
@@ -80,22 +80,35 @@ func InsertTests(db *sqlx.DB, tests []model.TestResult, upload model.Upload) err
 	return nil
 }
 
+func UpdateFlakyTests(db *sqlx.DB, commit model.Commit) error {
```

Comment:
Maybe only pass the commit.id into this method?

Reply:
Yes!